store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorage.java

```diff
@@ -66,7 +68,7 @@
     private Boolean isCacheEnabled = false;
     private GraphLibrary graphLibrary;
 
-    protected void startCacheServiceLoader() throws StorageException {
+    void startCacheServiceLoader() throws StorageException {
```

Comment:
package protected because FederatedStore is the only class that uses FederatedGraphStorage, it should not exist anywhere else or be accessed else where.

Reply:
Unless someone has extended the federated store cache and used the methods, in which case this could break that functionality. In the interests of limiting breaking changes, might it be better to leave this as-is?